.travis.yml

```diff
@@ -19,7 +19,7 @@ install: true # skips travis' default installation step which executes gradle as
 
 script:
   - ./gradlew clean build || travis_terminate 1
-  - ./docker-build/travis-docker-publish.sh || travis_terminate 1
+  - ./travis-build/travis-docker-publish.sh || travis_terminate 1
```

Comment:
1. you could also use travis' [build lifecycle step](https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle) `after_success` instead of all the `|| travis_terminate 1`

2. Do you really need to have a separate bash script file for it? Why not just:
````yaml
after_success: 
     - 'if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./gradlew pushDockerImage; fi'
````
like they also do it [here](https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions) in the travis docs as an example. This way you will not have any separate bash scripts anymore :)

Reply:
i tried this, but i didn't want to overload the travis config logic with the bash conditions, because actually (if we want to keep some helpful messages) it would look like:
```bash
if [[ "$TRAVIS_PULL_REQUEST" == "false" ]] ; then ./gradlew pushDockerImage; else echo "Pull request (\$TRAVIS_PULL_REQUEST != \"false\") identified, docker image build is skipped" ; exit 0 ; fi
```

which is not easy readable, thus i decided to carry out it to separate script.